cmd/ak/common/flags.go

```diff
@@ -68,3 +69,19 @@ func ToExitCodeWithSkipNotFoundFlag(cmd *cobra.Command, err error, whats ...stri
 	}
 	return exitErr
 }
+
+// maybe we could just run this check for all required flags instead of providing explicit ones?
+func UnsetEmptyFlags(cmd *cobra.Command, flags ...string) {
+	flagsMap := make(map[string]struct{}, len(flags))
+	for _, flag := range flags {
+		flagsMap[flag] = struct{}{}
+	}
+
```

Comment:
why is this needed?

Reply:
what? visit? or map?
I failed to understand your comment - plz see my answer for the previous one